website/docs/docs/running-a-dbt-project/command-line-interface/rpc.md

```diff
@@ -6,6 +6,10 @@ id: "rpc"
 ### Overview
 The `dbt rpc` command runs a Remote Procedure Call dbt Server. This server can compile and run queries in the context of a dbt project. Additionally, it provides methods that can be used to list and terminate running processes. The rpc server should be run from a directory which contains a dbt project. The server will compile the project into memory, then accept requests to operate against that project's dbt context.
 
+:::info
```

Comment:
Can you use a `<Callout>` here instead of `:::`? We need to upgrade our version of docusaurus to support the `:::` syntax :)

Usage looks like:
```
<Callout type='warning' title="Running on Windows">
  ...
</Callout>
```

via https://docs.getdbt.com/styles/

Reply:
Updated - thanks @drewbanin